homeassistant/components/insteon.py

```diff
@@ -30,15 +29,26 @@ def setup(hass, config):
     """
     if not validate_config(
             config,
-            {DOMAIN: [CONF_USERNAME, CONF_PASSWORD]},
+            {DOMAIN: [CONF_USERNAME, CONF_PASSWORD, CONF_API_KEY]},
             _LOGGER):
         return False
 
-    import insteon
+    try:
```

Comment:
Yeah, the isy994 component was written before the beefed up requirements handling was implemented.